src/_includes/feature_lists/cloud.njk

```diff
@@ -18,14 +18,14 @@
             info: "<p>Enable users to upgrade their Node-RED with a few clicks as new versions are released.</p><p>Our intuitive UI makes it clear when projects are running out-of-date stacks, so you have piece of mind over your flows running the latest and greatest versions.</p>"
         }, {
             id: "project-templates",
-            label: "Project Templates",
+            label: "Application Templates",
             values: ['check', 'check'],
             info: "<p>Spin up instances of Node-RED with all the relevant settings and Node-RED nodes already installed, customised to your own development environments.</p>"
         }, {
             id: "project-config",
-            label: "Individual Project Configuration",
+            label: "Individual Instance Configuration",
             values: ['check', 'check'],
-            info: "<p>Control all Node-RED settings on a project-by-project basis, set Environment Variables for deployment and precise configuration of how your Node-RED instances run.</p>"
+            info: "<p>Control all Node-RED settings on a application-by-application basis, set Environment Variables for deployment and precise configuration of how your Node-RED instances run.</p>"
```

Comment:
you've said "Individual Instance Configuration:, but then "application-by-application" basis, which one is it? Instance or Application? Both work imo, but should go either way, rather than mix.